sdk/iOS/OoyalaSkinSDK/OOSkinPlayerObserver.m

```diff
@@ -398,6 +400,22 @@ - (void)bridgeAudioTrackChangedNotification:(NSNotification *)notification {
   [self.viewController sendBridgeEventWithName:notification.name body:eventBody];
 }
 
+- (void)bridgePlaybackSpeedEnabledNotification:(NSNotification *)notification {
+  NSMutableDictionary *eventBody = [NSMutableDictionary new];
+
+  eventBody[@"playbackSpeedEnabled"] = @(self.player.isPlaybackSpeedEnabled);
+  eventBody[@"playbackSpeedRates"] = self.player.availablePlaybackSpeedRates;
```

Comment:
do we need the @ symbol here? You have it for the other two.

Reply:
Is this one a number as opposed to the one below being a string?

Reply:
@pilievOoyala It's a short syntax for ```NSNumber``` objects. Here it's a converting ```BOOL``` to a ```NSNumber```.  So, below it's an array.